python/pyspark/ml/feature.py

```diff
@@ -707,6 +855,34 @@ class CountVectorizerModel(JavaModel, _CountVectorizerParams, JavaMLReadable, Ja
     .. versionadded:: 1.6.0
     """
 
+    @since("3.0.0")
+    def setInputCol(self, value):
+        """
+        Sets the value of :py:attr:`inputCol`.
+        """
+        return self._set(inputCol=value)
+
+    @since("3.0.0")
+    def setOutputCol(self, value):
+        """
+        Sets the value of :py:attr:`outputCol`.
+        """
+        return self._set(outputCol=value)
+
+    @since("3.0.0")
+    def setMinTF(self, value):
+        """
+        Sets the value of :py:attr:`minTF`.
+        """
+        return self._set(minTF=value)
+
+    @since("3.0.0")
+    def setBinary(self, value):
```

Comment:
Ditto, [L923-L928](https://github.com/huaxingao/spark/blob/03c2e4a6c7581402de0c1067c31f5655d9992364/python/pyspark/ml/feature.py#L923-L928)

Reply:
The fix were in a few hours ago. Please let me know if you see other problems. Thanks! @zero323